app/models/event.rb

```diff
@@ -154,6 +154,17 @@ def compute_occupied_places
     application_letters.where(status: ApplicationLetter.statuses[:accepted]).count
   end
 
+  # locks the ability to change application statuses
+  def lock_application_status
+    application_status_locked = true
```

Comment:
you are right, I think this is a remnant of something we tried when we had a bug, it is now removed